public/index.html

```diff
@@ -15,6 +15,7 @@
 <body data-ng-view>
   <!-- TODO: Inject JS styles -->
   <script src="lib/angular/angular.js"></script>
+  <script src="lib/ng-file-upload/ng-file-upload-all.min.js"></script>
```

Comment:
Can you add this after `<script src="lib/ngInfiniteScroll/build/ng-infinite-scroll.js"></script>` ?